local/bin/py/build/configurations/pull_config_preview.yaml

```diff
@@ -86,24 +86,24 @@
           - "*/datadog_checks/*/__about__.py"
 
         - action: pull-and-push-file
-          branch: master
+          branch: alai97/legacy-integration-page-readme-cleanup
           globs:
           - docs/dev/README.md
           options:
             dest_path: '/developers/integrations/'
             file_name: '_index.md'
             front_matters:
-              dependencies: ["https://github.com/DataDog/integrations-core/blob/master/docs/dev/README.md"]
+              dependencies: ["https://github.com/DataDog/integrations-core/blob/alai97/legacy-integration-page-readme-cleanup/docs/dev/README.md"]
               
         - action: pull-and-push-folder
-          branch: master
+          branch: alai97/legacy-integration-page-readme-cleanup
```

Comment:
Reminder to change this one back as well

Reply:
```suggestion
          branch: master
```